wkhtmltopdf/tests/run.py

```diff
@@ -32,19 +52,11 @@
     WKHTMLTOPDF_DEBUG=True,
 )
 
-try:
-    django.setup()
-except AttributeError:
-    pass  # Django < 1.7; okay to ignore
-
-
-try:
-    from django.test.runner import DiscoverRunner
-except ImportError:
-    from discover_runner.runner import DiscoverRunner
+# Run tests with True debug settings (persistent temporary files).
+run_tests()
 
+settings.DEBUG = False
+settings.WKHTMLTOPDF_DEBUG = False
 
-test_runner = DiscoverRunner(verbosity=1)
-failures = test_runner.run_tests(['wkhtmltopdf'])
-if failures:
-    sys.exit(1)
+# Run tests with False debug settings to test temporary file delete operations.
+run_tests()
```

Comment:
I would run the tests with debug set to false only and use `override_settings` decorator where I want something specifically different.